### PR TITLE
[new release] codept 0.10.3

### PR DESCRIPTION
**Pull request by Octachron**
Codept 0.10.3 - Alternative ocaml dependency analyzer
-------
This is essentially a 4.08 compatibility release with a compiler upper bound on 4.09 since available data points show that codept generally breaks on new compiler releases.  

### Support

  * OCaml 4.08

### Features

* Flatter json and sexp output
* Less noisy ambiguous module warning

### Bug fixes

  * Missing dependencies while matching first-class modules

### CODE REVIEW

File: packages/codept/codept.0.10.3/opam [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+author: "octachron <octa@polychoron.fr>"
+maintainer: "octachron <octa@polychoron.fr>"
+homepage: "https://github.com/Octachron/codept"
+bug-reports: "https://github.com/Octachron/codept/issues"
+license: "gpl-3"
+dev-repo: "git+https://github.com/Octachron/codept.git"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+  ["dune" "build" "@private-docs"] {with-doc}
```
Comment (mseri) [inline packages/codept/codept.0.10.3/opam]: Are you sure you want to bundle the private docs to the documentation?

Comment (Octachron) [reply packages/codept/codept.0.10.3/opam]: Right, it is probably better to remove it for now.

Comment (Octachron) [reply packages/codept/codept.0.10.3/opam]: I have removed this line (and the odoc dependency).

**Comment by mseri** [comment]
Thanks

**Pull request merged**